lib/router/src/cache.rs

```diff
@@ -2,10 +2,10 @@ use std::{hash::Hash, time::Duration};
 
 use futures::{try_ready, Async, Future, Poll, Stream};
 use indexmap::IndexMap;
-use log::debug;
+
 use tokio::sync::lock::Lock;
 use tokio_timer::{delay_queue, DelayQueue, Error, Interval};
-
+use tokio_trace::debug;
```

Comment:
Nit: Can we keep a newline between the use declarations and the doc comments?

Reply:
I think this is rustfmt, but I'll double check.